includes/class-sensei-course.php

```diff
@@ -3652,6 +3658,7 @@ public function is_legacy_course( $course ) {
 
 		$course_blocks = [
 			'sensei-lms/course-outline',
+			'sensei-lms/course-progress',
```

Comment:
Since we also removed the action for the `Take this course` in this PR, I think we should also add it to the check, right?

Reply:
Yep! Fixed in 3a88e111aee351a5a2a8455d7bce628660ba7336